client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -144,15 +180,26 @@ class EditorMediaModalDetailItem extends Component {
 		);
 	}
 
-	renderImageEditorButtons( item, classname = 'is-desktop' ) {
-		const { site } = this.props;
-
-		if ( ! enableImageEditing( item, site ) ) {
+	renderMediaEditorButtons( item, classname = 'is-desktop' ) {
+		if ( ! item ) {
 			return null;
 		}
 
+		const mimePrefix = MediaUtils.getMimePrefix( item );
 		const classes = classNames( 'editor-media-modal-detail__edition-bar', classname );
 
+		return 'video' === mimePrefix
+			? this.renderVideoEditorButtons( item, classes )
+			: this.renderImageEditorButtons( classes, classname );
```

Comment:
It's a bit awkward what we're doing with `classes` here (including this signature 😛). What about moving [R189](https://github.com/Automattic/wp-calypso/pull/11653/files#diff-ee87f4ffa81441c89dbef5404a1de352R189) to `renderVideoEditorButtons` and `renderImageEditorButtons`, respectively (thus duplicating that LOC)? That'd allow us to drop the `classes` arg (and pass only `classname` in both cases).